### ISSUE TITLE
Add Detail Level Parameters

### ISSUE THREAD

**Issue by TrimarcJake** [TrimarcJake/Locksmith#167]
**Is your feature request related to a problem? Please describe it:**
Locksmith is capable of spitting out tons of data about identified issues, but that functionality is hard to discover. 

**Describe the solution or enhancement you would like:**
Locksmith should include parameters to enable different detail levels. For example,
- `-Minimal` might align with the output from the current `-Mode 0`
- `-Detailed` would be similar to `-Mode 1`
- `-Full` would be like `-Mode 1` but include revert code along with fixes
- `-ELI5` would include all detail shown in `-Full` plus explanations on what the issue means and what the fix script does

**Describe alternatives you've considered:**
The alternative is a training course in AD CS security.

**Additional context:**
@techspence has good ideas!

**Comment by jakehildreth** [comment]
Good idea... for Locksmith 2!
